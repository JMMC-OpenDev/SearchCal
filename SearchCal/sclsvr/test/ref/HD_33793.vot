version https://git-lfs.github.com/spec/v1
oid sha256:ac12b99fca4cc4c0b25e5f51bc02b871eef98ed99739106587189cff69f0f213
size 241234
