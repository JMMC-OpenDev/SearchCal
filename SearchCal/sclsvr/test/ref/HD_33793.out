version https://git-lfs.github.com/spec/v1
oid sha256:d22c742c1977a907eb3431db9835ac77a03313375054dee538c0e5c36cdd75f7
size 377793
